.gitignore

```diff
@@ -44,8 +44,15 @@ app.*.map.json
 /android/app/profile
 /android/app/release
 
+# lock
+pubspec.lock
+
 # env 
-.env*
+.env
+
+# Firebase
+/android/app/google-services.json
+/ios/Runner/GoogleService-Info.plist
```

Comment:
We should add a template for these files

Reply:
These files are entirely generated by Firebase, so I don't think we need to do so